File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/boxplotcharts/LocalDataBindingController.java
package com.kendoui.spring.controllers.boxplotcharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-box_plot_chart-local_data-controller")
@RequestMapping(value="/box-plot-charts/")
public class LocalDataBindingController {
    @RequestMapping(value = "/local-data-binding", method = RequestMethod.GET)
    public String index(Model model) {
        model.addAttribute("ozoneConcentration", ChartDataRepository.OzoneConcentration());
     
        return "/box-plot-charts/local-data-binding";
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/Box_Plot_Charts/Local_Data_BindingController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Box_Plot_ChartsController : Controller
    {
        public ActionResult Local_Data_Binding()
        {
            return View(ChartDataRepository.OzoneConcentration());
        }
    }
}